public/src/components/epicTests/epicTestsForm.tsx

```diff
@@ -138,26 +164,91 @@ class EpicTestsForm extends React.Component<EpicTestFormProps, EpicTestsFormStat
       })
   }
 
+  requestEpicTestsLock = () => {
+    requestLock(FrontendSettingsType.epicTests).then(
+      response => {
+        response.ok ? this.setState({ editMode: true }) : alert("Error - can't request lock!");
```

Comment:
thinking about this again, it would be better to always call `this.fetchStateFromServer()` when these requests come back with `response.ok` and set `editMode` elsewhere.
I'll create a PR with a suggested change